src/ploomber/util/markup.py

```diff
@@ -23,15 +23,15 @@ def block_code(self, code, lang=None):
         return markdown(md)
     else:
 
-        class HighlightRenderer(mistune.Renderer):
+        class HighlightRenderer(mistune.HTMLRenderer):
```

Comment:
remove these changes and merge from master, I've already fixed this: https://github.com/ploomber/ploomber/commit/8acdf9a10eb9e207f14061c283a7450053ba8833

then I can merge and see if that triggers the action

Reply:
Sure!